### PR TITLE
Propose v0.1.4

### PR DESCRIPTION
**Pull request by suesslin**
README.md has been updated for the latest version, which would be all this PR features.
New features would be:

* custom_encipher
* custom_decipher

Also, encipher & decipher were updated to use custom_encipher and custom_decipher, with the roman/latin alphabet being passed in as base.

### CODE REVIEW

**Comment by Natim** [comment]
This is not a patch change because you are changing the API. It should be a major change stated as `1.0.0` or at last `0.2.0` if you think the API is not stabilized yet and might still change.

More info about that there: http://semver.org/

**Comment by suesslin** [comment]
Thank you for the reference! I'll keep that in mind. Learning something new every day 😊

**Comment by luki** [comment]
@Natim Everything good for 0.2.0? (:

**Review by suesslin — COMMENTED** [review]
@Natim Everything good for 0.2.0 release?

File: tests/caesarlib.rs [hunk 1]
```diff
@@ -55,8 +55,8 @@ mod tests {
     #[test]
     fn custom_base_sequence_correct_encipherd_and_deciphered() {
         let base = "zyxwvutsrqponmlkjihgfedcbaZYXWVUTSRQPONMLKJIHGFEDCBA";
-        let offset = 5;
         let test_word = "FooBar";
+        let offset = 5;
```
Comment (Natim) [inline tests/caesarlib.rs:59]: Why this change?

Comment (suesslin) [reply tests/caesarlib.rs:59]: Conflict occurred

Comment (suesslin) [reply tests/caesarlib.rs:59]: Is it good for release,?

Comment (Natim) [reply tests/caesarlib.rs:59]: Yes I guess it is good for the release.
I put them in the argument order (base, offset, test_word) which is also the alphabetical order, but it doesn't really matter I guess.

Comment (suesslin) [reply tests/caesarlib.rs:59]: @Natim It is a detail that's nice. Before, I tried putting it in a 'logical' order.
Also, I was answering from school, so I couldn't really have an exact look at everything that happened!
